doc/source/changelog.rst

```diff
@@ -0,0 +1,922 @@
+.. _ref_release_notes:
+
+Release notes
+#############
+
+This document contains the release notes for the project.
+
+.. vale off
```

Comment:
@klmcadams  should we turn on vale at the end?

Reply:
Ohh yeah, it should be right after the towncrier release notes start line. I will add it

Reply:
You're right, it should be at the end. I was referring to the setup guide oops